### PR TITLE
BGE: disable save button to prevent duplicate batches

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- Disable Batch save button after it's clicked

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by bethbrains** [comment]
** lurch: attach W-029655

**Comment by bethbrains** [comment]
**lurch: attach W-029655

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fPPCQA2" target="_blank">W-029655</a>

File: src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js [hunk 1]
```diff
@@ -887,6 +892,15 @@
                 this.onMetadataUpdated.notify();
             }
 
+            /**
+             * @description sets the pendingsave flag to disable Save button so duplicates can't be created
+             * @return void.
+             */
+            function togglePendingSave() {
+                this.pendingSave = !this.pendingSave;
```
Comment (lmeerkatz) [inline src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js]: Would it be safer to explicitly set pendingSave to true or false? Is there a condition like clicking twice that could cause the state to toggle back to false incorrectly?

**Comment by ptewson-sfdo** [comment]
Not blocking this feature, but noted that errors from the first page (blank values for expected total for instance) don't surface to user at the save state - so save fails without a visible explanation. However, you can navigate back and fix the issue if you figure out what's going on, then return to the end and save.

**Comment by bethbrains** [comment]
that scenario is fixed in another branch :)

**Pull request merged**